packages/@guardian/source-react-components-development-kitchen/src/editorial-button/styles.ts

```diff
@@ -160,6 +161,10 @@ export const decideFont = (
 					return css`
 						color: ${specialReport[400]};
 					`;
+				case ArticleSpecial.SpecialReportAlt:
+					return css`
+						color: ${palette.specialReportAlt[200]};
```

Comment:
@ioannakok @jamie-lynch @HarryFischer Is this what you expected to see? Please also see the quote icon as well